src/Admin/SiteHealth.php

```diff
@@ -208,21 +302,137 @@ static function ( $available_service ) {
 			$description .= '</p>';
 		}
 
+		if ( $is_using_object_cache ) {
+			$status = 'good';
+			$color  = 'green';
+			$label  = __( 'Persistent object caching is enabled', 'amp' );
+		} elseif ( $has_page_caching ) {
+			$status = 'good';
+			$color  = 'blue';
+			$label  = __( 'Persistent object caching is not enabled, but page caching was detected', 'amp' );
+		} else {
+			$status = 'recommended';
+			$color  = 'orange';
+			$label  = __( 'Persistent object caching is not enabled', 'amp' );
+		}
+
 		return [
 			'badge'       => [
 				'label' => $this->get_badge_label(),
-				'color' => $is_using_object_cache ? 'green' : 'orange',
+				'color' => $color,
 			],
 			'description' => wp_kses_post( $description ),
 			'actions'     => $this->get_persistent_object_cache_learn_more_action(),
 			'test'        => 'amp_persistent_object_cache',
-			'status'      => $is_using_object_cache ? 'good' : 'recommended',
-			'label'       => $is_using_object_cache
-				? esc_html__( 'Persistent object caching is enabled', 'amp' )
-				: esc_html__( 'Persistent object caching is not enabled', 'amp' ),
+			'status'      => $status,
+			'label'       => $label,
+		];
+	}
+
+	/**
+	 * Get the test result data for whether there is page caching or not.
+	 *
+	 * @return array
+	 */
+	public function page_cache() {
+		$has_page_caching = $this->has_page_caching();
+
+		set_transient( self::HAS_PAGE_CACHING_TRANSIENT_KEY, true === $has_page_caching, MONTH_IN_SECONDS );
+
+		$badge_color = 'orange';
+		$status      = 'recommended';
+		$label       = __( 'Page caching is not detected', 'amp' );
+
+		$description = '<p>' . esc_html__( 'The AMP plugin performs at its best when page caching is enabled. This is because the additional optimizations performed require additional server processing time, and page caching ensures that responses are served quickly.', 'amp' ) . '</p>';
+
+		/* translators: 1 is Cache-Control, 2 is Expires, and 3 is Age */
+		$description .= '<p>' . sprintf( __( 'Page caching is detected by making three requests to the homepage and looking for %1$s, %2$s, and/or %3$s HTTP response headers.', 'amp' ), '<code>Cache-Control: max-age=…</code>', '<code>Expires</code>', '<code>Age</code>' );
+
+		if ( is_wp_error( $has_page_caching ) ) {
+			$error_info = sprintf(
+				/* translators: 1 is error message, 2 is error code */
+				__( 'Unable to detect page caching due to possible loopback request problem. Error: %1$s (Code: %2$s)', 'amp' ),
```

Comment:
Could maybe also recommend checking the loopback request site health as well.

Reply:
Like so?

```suggestion
				__( 'Unable to detect page caching due to possible loopback request problem. Please check the loopback test results. Error: %1$s (Code: %2$s)', 'amp' ),
```

Reply:
Hmm they'd only need to check if the loopback test is failing so maybe something like:

```suggestion
				__( 'Unable to detect page caching due to possible loopback request problem. Please check if the loopback site health test is passing. Error: %1$s (Code: %2$s)', 'amp' ),
```

Reply:
To further iterate on that:

```suggestion
				__( 'Unable to detect page caching due to possible loopback request problem. Please verify that the loopback request test is passing. Error: %1$s (Code: %2$s)', 'amp' ),
```

Reply:
Perfect.